stages/apps/sharedsvc/outputs.tf

```diff
@@ -38,6 +38,10 @@ output "sonarqube_collector_url" {
   value = "http://rode-collector-sonarqube.${module.rode_namespace.name}.svc.cluster.local/webhook/event"
 }
 
+output "harbor_collector_url" {
+  value = "http://rode-collector-harbor.${module.rode_namespace.name}.svc.cluster.local/webhook/event"
```

Comment:
Why are these not https?  Just curious.

Reply:
It's all traffic inside the Kubernetes cluster, neither of these collectors are exposed externally. 

If we did want that, we could update the collectors to serve HTTPS or use something like Istio to get mTLS

Reply:
Something for later thanks!